editor/components/block-mover/index.js

```diff
@@ -49,32 +35,42 @@ export function BlockMover( { onMoveUp, onMoveDown, isFirst, isLast, uids, block
 				className="editor-block-mover__control"
 				onClick={ isFirst ? null : onMoveUp }
 				icon={ upArrow }
-				tooltip={ __( 'Move Up' ) }
-				label={ getBlockMoverLabel(
-					uids.length,
-					blockType && blockType.title,
-					firstIndex,
-					isFirst,
-					isLast,
-					-1,
-				) }
+				label={ __( 'Move up' ) }
+				aria-describedby="editor-block-mover__up-description"
 				aria-disabled={ isFirst }
 			/>
 			<IconButton
 				className="editor-block-mover__control"
 				onClick={ isLast ? null : onMoveDown }
 				icon={ downArrow }
-				tooltip={ __( 'Move Down' ) }
-				label={ getBlockMoverLabel(
-					uids.length,
-					blockType && blockType.title,
-					firstIndex,
-					isFirst,
-					isLast,
-					1,
-				) }
+				label={ __( 'Move down' ) }
+				aria-describedby="editor-block-mover__down-description"
 				aria-disabled={ isLast }
 			/>
+			<span id="editor-block-mover__up-description" className="editor-block-mover__description">
```

Comment:
Don't we have a specific className for these kind of labels? like `reader-text` or something?

Reply:
Yes there's `screen-reader-text` but I'd like the spans not to be read out as normal content. Hiding them with `display: none` prevents that and still works for `aria-describedby`.